src/utilities/sql/SqlFile.cpp

```diff
@@ -1385,4 +1385,28 @@ bool SqlFile::hasIlluminanceMapYear() const {
   return true;
 }
 
+boost::optional<double> SqlFile::assemblyUFactor(const std::string& rowName) const {
```

Comment:
I get why you use rowName here, but I wonder if it wouldn't be clearer to name the param "subSurfaceName" or "assmemblyName" or something?